build/charts/antrea/conf/antrea-agent.conf

```diff
@@ -306,6 +306,11 @@ kubeAPIServerOverride: {{ .Values.kubeAPIServerOverride | quote }}
 # 10.96.0.10:53, [fd00:10:96::a]:53).
 dnsServerOverride: {{ .Values.dnsServerOverride | quote }}
 
+# The fqdnCacheMinTTL setting helps address the problem of applications caching DNS response IPs beyond the TTL value for the DNS record.
```

Comment:
s/The fqdnCacheMinTTL setting helps address/fqdnCacheMinTTL helps address

Reply:
@hkiiita not addressed correctly, the current sentence is not grammatically correct